### DISCUSSION TITLE
README notes a feature in 1.3.2 or later...

### DISCUSSION THREAD

**Discussion by sckirk** [spring-projects/spring-retry#272] [Q&A] [answered]
I am super excited for this feature described in the README: `Version 1.3.2 and later supports matching a parameterized (generic) return type to detect the correct recovery method:`

When will version 1.3.2 be available?

Could the README be updated so that others aren't disappointed getting excited about a feature that doesn't exist yet?

**Comment by snicoll** [comment, marked as answer]
Thanks for the interest @sckirk. We'll release 1.3.2 shortly as we've configured a new release pipeline and we'd like to give it a try.

As for this particular feature, I can see that the fix was actually merged in `1.3.1` and the readme wrongly references 1.3.2. So you should be able to use this already.

**Reply by garyrussell** [reply]
@snicoll; thanks for configuring the new pipeline and releasing 1.3.2; actually the feature missed 1.3.1; while the commit was authored in 2020, I only came across it and committed it in 2021, after 1.3.1...

```
% git log -4 --pretty=fuller ae3ad4a249382b198947c09e514a0d83c390c2ea
commit ae3ad4a249382b198947c09e514a0d83c390c2ea
Author:     Gary Russell <grussell@vmware.com>
AuthorDate: Tue Mar 9 16:46:47 2021 -0500
Commit:     Gary Russell <grussell@vmware.com>
CommitDate: Tue Mar 9 16:48:21 2021 -0500

    Docs and test polishing for previous commit

commit 4d2bb2be27f668e5b6b5b5c6699658dce2ac52da
Author:     deepesh-verma <deepeshvrm1@gmail.com>
AuthorDate: Fri Sep 4 01:10:46 2020 +0530
Commit:     Gary Russell <grussell@vmware.com>
CommitDate: Tue Mar 9 16:47:01 2021 -0500

    Add generic check for function return type
    
    Fixes: [#215](https://github.com/spring-projects/spring-retry/issues/215)
    Signed-off-by: deepesh-verma <deepeshvrm1@gmail.com>

commit c452ec3237c1310b2910ce33be72a797ef1562c9
Author:     Dave Syer <dsyer@vmware.com>
AuthorDate: Fri Jan 8 17:41:04 2021 +0000
Commit:     Dave Syer <dsyer@vmware.com>
CommitDate: Fri Jan 8 17:43:43 2021 +0000

    Revert to snapshots

commit 2e048d7621376adecfb17eac24cee34a11ad44c7 (tag: 1.3.1.RELEASE)
Author:     Dave Syer <dsyer@vmware.com>
AuthorDate: Fri Jan 8 17:39:23 2021 +0000
Commit:     Dave Syer <dsyer@vmware.com>
CommitDate: Fri Jan 8 17:42:35 2021 +0000

    Bump for 1.3.1
```

**Reply by snicoll** [reply]
Ooops. OK I'll revert the readme update then.
